src/models/club.ts

```diff
@@ -51,13 +52,21 @@ namespace Club {
 		domain: string
 	}
 
+	/**
+	 * PublicClubInfo contains the content for a public ClubHub website.
+	 */
+	export interface PublicClubInfo {
```

Comment:
Shouldn't this be appearing in a d.ts file somewhere? Do you need to run `make build`?